eskip/example_test.go

```diff
@@ -38,12 +39,15 @@ func Example() {
 
 		// route definition with a shunt (no backend address)
 		route2: Path("/some/other/path") -> static("/", "/var/www") -> <shunt>;
-		
+
 		// route definition directing requests to an api endpoint
 		route3:
 			Method("POST") && Path("/api") ->
 			requestHeader("X-Type", "ajax-post") ->
-			"https://api.example.org"
+			"https://api.example.org";
+
+		// route definition with a loopback (no backend address)
+		route4: Path("/some/other/path") -> static("/", "/var/www") -> <loopback>;
```

Comment:
serve static with a loopback? Interesting example. What does this actually do? Maybe better to have a `setHeader` in the example to make it more meaningful